package.json

```diff
@@ -48,6 +48,8 @@
     "danger": "danger",
     "deploy": "lerna run --scope terra-core-site deploy",
     "dependency-markdown": "node scripts/dependency-markdown-generator/DependencyMarkdownGenerator.js",
+    "generate-config": "node node_modules/terra-dev-site/scripts/generate-component-config/generate-component-config.js -s 'packages/terra-site/src/examples/*/Index.jsx' -s 'packages/terra-*/tests/{nightwatch,nightwatch/*}/!(*-spec|*Tests|*TestRoutes|*Template|common|MockComponent|SignatureConstants)+(.jsx|.js)' -s 'packages/terra-table/tests/nightwatch/*/components/!(*-spec|*Tests|*TestRoutes|*Template|common|MockComponent|SignatureConstants)+(.jsx|.js)'",
+    "generate-config": "generate-config -s 'packages/terra-site/src/examples/*/Index.jsx' -s 'packages/terra-*/tests/{nightwatch,nightwatch/*}/!(*-spec|*Tests|*TestRoutes|*Template|common|MockComponent|SignatureConstants)+(.jsx|.js)' -s 'packages/terra-table/tests/nightwatch/*/components/!(*-spec|*Tests|*TestRoutes|*Template|common|MockComponent|SignatureConstants)+(.jsx|.js)'",
```

Comment:
Are both of these scripts supposed to have the same names?

Reply:
😢 Nope. Need to remove the first